docs/node/daemon/config/settings.mdx

```diff
@@ -317,6 +369,35 @@ max_tx_bytes = 4194304
 
 # Size of the cache (used to filter transactions we saw earlier) in transactions
 cache_size = 10000
+
+#######################################################
+###         State Sync Configuration Options        ###
+#######################################################
+[chain.statesync]
+# State sync rapidly bootstraps a new node by discovering, fetching, and restoring the database
+# snapshot from peers instead of fetching and replaying historical blocks. Requires some peers in
+# the network to take and serve snapshots. State sync is not attempted if the node
+# has any local state (LastBlockHeight > 0). The node will have a truncated block history,
+# starting from the height of the snapshot.
+enable = true
+
+# Path to the directory where the received snapshot is stored
+snapshot_dir = "rcvd_snaps"
+
+# Trusted snapshot providers (comma-separated chain RPC servers) are the source-of-truth for the snapshot integrity.
+# Snapshots are accepted for statesync only after verifying it with these trusted snapshot providers.
+# Atleast 2 trusted rpc servers are required for enabling state sync.
```

Comment:
"Atleast" => "At least"

---

Can the comment describe what is being trusted, what is the truth, and what is verified? (Hashes, heights, etc)